.travis.yml

```diff
@@ -18,10 +18,10 @@ matrix:
       env: TARGET=x86_64-unknown-linux-musl
     # Minimum Rust supported channel.
     - os: linux
-      rust: 1.28.0
+      rust: 1.32.0
       env: TARGET=x86_64-unknown-linux-gnu
     - os: linux
-      rust: 1.28.0
+      rust: 1.32.0
```

Comment:
I would just go ahead and bump this up to 1.34.0.

Reply:
Okay will do